app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.skill_level = skill_level
```

Comment:
The `skill_level` attribute should be stored as a protected attribute, i.e., `_skill_level`, to align with the task requirement of encapsulating additional parameters as protected attributes.